policies/travel-policy.md

```diff
@@ -40,14 +38,15 @@ Definitions
     -   Accommodations while attending meetings, conferences, etc.
     -   Meals.
 -   **Non-reimbursable** expenses: are expenses that OpenSSL will not
-    reimburse and are at the Traveller's own cost some examples of these
+    reimburse and are at the Traveller's own cost. Some examples of these
     are (this is not an exhaustive list):
     -   Costs of entertainment, including amusement, diversion, and social
     activities and any costs directly associated with such costs (such
     as tickets to shows or sports events, meals, lodging, rentals,
     transportation, and gratuities)
-    -   Tips except when it's already been included in the bill as a service
-    charge;
+    -   Tips except reasonable tips in keeping with the expectation for the
+    country being visited can be claimed, or any service charges included in
+    the bill.
```

Comment:
This list is supposed to be an example list of things that can't be claimed. But this entry is quite confusing because it mostly talks about things that can be claimed. I would remove this bullet point completely. Tips isn't a good example of things that can't be claimed, and the meals section already discusses tips.

Reply:
Removed the section related to tips